src/sage/structure/formal_sum.py

```diff
@@ -95,6 +94,11 @@ def __init__(self, x, parent=None, check=True, reduce=True):
         - ``reduce`` -- reduce (default: ``True``) if ``False``, do not
           combine common terms
 
+      .. WARNING::
+
+        Setting ``reduce`` to ``False`` can cause issues when comparing
+        equal sums but with different orders and/or cancellations.
```

Comment:
```suggestion
        .. WARNING::

            Setting ``reduce`` to ``False`` can cause issues when comparing
            equal sums but with different orders and/or cancellations.
```
The alignment is off here.